pkg/cmd/cli/cmd/rollback.go

```diff
@@ -221,12 +221,13 @@ func (o *RollbackOptions) Run() error {
 
 	// If this is a dry run, print and exit.
 	if o.DryRun {
-		describer := describe.NewDeploymentConfigDescriberForConfig(o.oc, o.kc, newConfig)
-		description, err := describer.Describe(newConfig.Namespace, newConfig.Name)
+		// Compare against the current deployment config
+		configName := deployutil.DeploymentConfigNameFor(target)
+		currentConfig, err := o.oc.DeploymentConfigs(o.Namespace).Get(configName)
 		if err != nil {
 			return err
 		}
-		o.out.Write([]byte(description))
+		o.out.Write([]byte(kutil.ObjectGoPrintSideBySide(currentConfig, newConfig)))
```

Comment:
I believe we can break down the previous human readable description into lines, and prefix those that differ between the two configs with `+` and `-`. How does that sound?